build.gradle

```diff
@@ -85,6 +85,10 @@ repositories {
     maven { url "https://plugins.gradle.org/m2/" }
 }
 
+dependencies {
```

Comment:
@rursprung do we really need it?

Reply:
it doesn't compile without. but maybe this is instead something which could/should be fixed in opensearch directly rather than the plugin?

Reply:
Hm ... I don't see any compilation issues by removing that, could you please share the failure?

Reply:
very weird, i had previously had these errors (still had them in my scrollback, luckily):
```
> Task :compileTestJava
...\opensearch-plugin-template-java\src\test\java\org\opensearch\path\to\plugin\RenamePluginIT.java:11: error: package org.apache.hc.core5.http does not exist
import org.apache.hc.core5.http.ParseException;
                               ^
...\opensearch-plugin-template-java\src\test\java\org\opensearch\path\to\plugin\RenamePluginIT.java:12: error: package org.apache.hc.core5.http.io.entity does not exist
import org.apache.hc.core5.http.io.entity.EntityUtils;
```

i don't know why, but now it even compiles like this... 😕
all the better then!